doc/modules/recommended_practices.rst

```diff
@@ -0,0 +1,30 @@
+=====================
+Recommended Practices
```

Comment:
Maybe we could name this section even more explicitly such as "Common methodological pitfalls".

Most of the rest of the scikit-learn doc is about "recommended practices" so I would rather have more explicit for this section.